src/simulation/simulation_results.jl

```diff
@@ -1,6 +1,7 @@
 function check_folder_integrity(folder::String)
     folder_files = readdir(folder)
     alien_files = setdiff(folder_files, KNOWN_SIMULATION_PATHS)
+    alien_files = filter(x -> !any(occursin.(IGNORABLE_FILES, x)), alien_files)
```

Comment:
This has caused complaints for years. Who cares if there are extra files in the directory? Can we remove this altogether?

Reply:
Looks like it got added [here](https://github.com/NREL-Sienna/PowerSimulations.jl/commit/58d6e94e227b02ed4540969b0c6e4d0f147757e4), can we remember the rationale? @jd-lara